contrib/config_option_database/config_database.py

```diff
@@ -39,18 +39,67 @@ def parse_args():
     return args
 
 
+def merge_block_stored_as_an_option(keyword, arguments, options, blocks):
+    options.remove((keyword, arguments))
+    if ('', arguments) not in blocks[keyword]['options']:
+        blocks[keyword]['options'].append(('', arguments))
+
+
+def is_option_with_multiple_signature(option):
+    _, arguments_list = option
+    return len(arguments_list) > 1
+
+
+def merge_mergeable_options(mergeable_options, options):
+    for keyword, arguments_list in mergeable_options:
+        merged_arguments = tuple()
+        for arguments in sorted(arguments_list):
+            if merged_arguments:
+                merged_arguments += ('|',)
```

Comment:
[optional]
My assumption upon checking the database(after the change):
* sequence is `"foo": [ <string> <number> ]`
* options is `"foo": [<string> | <number>]`

Before change:
* sequence is the same
* options is `"foo": [<string> ], "foo": [ <number>]`

Would it be possible not to change the database itself ? And provide the `|` in the view itself.
I like the new view, just not so much the database change.

Reply:
@mitzkia You don't need the database change just the view change, right?

Reply:
@kokan Yes, it is completely possible :)

Reply:
@gaborznagy  The view for a human is great, also the other changes are welcomed like listing.

Reply:
Pushed some new commits, the merging of options with multiple signatures is now only done at the printing stage.